client/src/components/NavbarComponents.vue

```diff
@@ -154,7 +154,8 @@ const menu = [
   { icon: 'mdi-account-group', title: 'Team', link: '/teams' },
   { icon: 'mdi-calendar', title: 'Events', link: '/events' },
   { icon: 'mdi-trophy', title: 'Challenges', link: '/challenge' },
-  { icon: 'mdi-star', title: 'Leaderboards', link: '/' }
+  { icon: 'mdi-star', title: 'Leaderboards', link: '/' },
+  { icon: 'mdi-currency-usd', title: 'Donate', link: 'https://stride-for-education.raisely.com' }
```

Comment:
Because it is linking to an external website we have to render it separately outside of the v-list-item v-for loop.
`:to` is only used for internal routing